### PR TITLE
Landing Page

### PR DESCRIPTION
**Pull request by shrey27**
Features
- Landing page
- Top categories for the project
- CTA buttons for redirection

### CODE REVIEW

File: src/frontend/utility/footer/footer.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  padding: 0.2rem;
+  width: 2rem;
+  height: auto;
+  object-fit: contain;
+}
+.fixed {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
```
Comment (samyakshah3008) [inline src/frontend/utility/footer/footer.css:21]: You can use shorthand property for less lines of code

**Review by samyakshah3008 — APPROVED** [review]
You have really good folder structure 
Clean code and amazing UI 
Loved how you documented in PR Preview

Suggestion: You can add some necessary comments in your code for better review experience
Also, one optional suggestion: You can remove white background of your footer logo icons 

Rest is all good for approval ✅

File: src/frontend/utility/footer/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+export function Footer({ fixed }) {
+  return (
+    <footer
+      className={`footer--ctr flex-ct-ct flex-vertical sm-s ${
+        fixed && 'fixed'
+      }`}
+    >
+      <span className='md bl mg-half'>Connect With Me</span>
+      <div>
+        <a href='https://twitter.com/home'>
```
Comment (Shah-Arsalan) [inline src/frontend/utility/footer/index.js:12]: remove a tag from your code

**Review by Shah-Arsalan — APPROVED** [review]
The overall code looks  nice . Found that you used variables in Routes for specifying path . Going to implement same in my code as well

File: src/frontend/utility/footer/index.js (hunk 2 above)
Comment (shrey27) [reply src/frontend/utility/footer/index.js:12]: yes they will be replaced with Link component

File: src/frontend/utility/footer/footer.css (hunk 1 above)
Comment (shrey27) [reply src/frontend/utility/footer/footer.css:21]: yeah I will do that

**Pull request merged**